### ISSUE TITLE
Add filter for extension author credit link

### ISSUE THREAD

**Issue by johnalarcon** [codepotent/Update-Manager#45]
# Description
Add a filter that allows extension authors to append their own credit link to the view(s) created by their extension(s).

# Context
The desire is to keep the interface consistent.

**Comment by johnalarcon** [comment]
This has been implemented and can be used as follows in the next release:

```
add_filter('codepotent_update_manager_extension_footer_{your-slug-name}', 'some_function_name');
function some_function_name($text) {
    return 'Featuring <a href="#">My Cool Extension</a> by <a href="#">Author Name</a>';
}
```
**Note**: all HTML is stripped except for the &lt;a> tag which can use both the `href` and `title` attributes.

**Comment by johnalarcon** [comment]
Implemented on the milestone-2.0.0 branch.
